release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -538,9 +538,50 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-insights-operator-enhancements_{context}"]
 === Insights Operator
 
-* The Insights Operator now collects the `haproxy_exporter_server_threshold` metric. (link:https://issues.redhat.com/browse/OCPBUGS-36687[*OCPBUGS-36687*])
+// Engineering reference CCXDEV-12582
 
-* Previously, the Insights Operator gathered information about all Ingress Controller certificates, including their `NotBefore` and `NotAfter` dates. This data is now compiled into a `JSON` file located at `aggregated/ingress_controllers_certs.json` for easier monitoring of certificate validity across the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-35727[*OCPBUGS-35727*])
+The Insights Operator now collects more {product-title} container log data from namespaces prefixed with either the  `openshift-` or `kube-` prefix and generates recommendations much faster.
+Enhancements have also been made to give you more flexibility in how the data to be collected gets defined for your service.
+
+==== Rapid Recommendations
+
+This release introduces a new feature called Rapid Recommendations, which provides a more dynamic and version-independent mechanism for remotely configuring the rules that determine which data the Insights Operator collects.
+
+Rapid Recommendations builds on the existing conditional data gathering mechanism.
+The Insights Operator connects to a secure remote endpoint service running on `/console.redhat.com` to retrieve definitions that contain the rules for determining which container log messages are filtered and collected by Red Hat.
+
+The conditional data-gathering definitions, also referred to as rules, get configured through an attribute named `conditionalGathererEndpoint` in the link:https://github.com/openshift/insights-operator/blob/master/config/pod.yaml[`pod.yml`] configuration file.
+
+
+```bash
+[source,bash]
```

Comment:
Hi @michelle-purcell . JUst one change here as the `source,bash` should not show in the preview. You can drop the ```bash and ``` delimiters. This should fix this issue. 

![Screenshot from 2024-10-23 17-29-58](https://github.com/user-attachments/assets/7a9ee7e7-12a5-4c95-85cc-cbb7c2cde029)